http_request/http_request.py

```diff
@@ -105,6 +101,33 @@ def addKeyToObject(self, key="", msg="", data={}):
     def formatJson(self, data):
         return str(data).strip("'<>() ").replace("'", '"')
 
+    def redaction(self, msg):
+        REDACTED = '[REDACTED]'
+
+        msg = str(msg)
+
+        try:
+            # Using MONGO_URI
```

Comment:
please either change or remove these comments (for our future selves). backwork-notifier-http has no notion of MONGO_URI or MONGO_HOST